code/web/src/modules/StyleSurvey/StyleSurvey.js

```diff
@@ -155,8 +156,8 @@ class StyleSurvey extends Component {
 						data-testid={`${image.id}-image`}
 						key={i}
 						data-category={image.style}
-						width={100}
-           				height={250}
+						width={'100%'}
```

Comment:
I noticed this change back to a string, I changed it to a number because of the lint error that I talked about in my last PR that closes ticket #45